homeassistant/components/comelit/coordinator.py

```diff
@@ -30,6 +35,38 @@ def __init__(self, hass: HomeAssistant, host: str, pin: int) -> None:
             name=f"{DOMAIN}-{host}-coordinator",
             update_interval=timedelta(seconds=5),
         )
+        device_registry = dr.async_get(self.hass)
+        device_registry.async_get_or_create(
+            config_entry_id=self.config_entry.entry_id,
+            identifiers={(DOMAIN, self.config_entry.entry_id)},
+            model=BRIDGE,
+            name=f"{BRIDGE} ({self.api.host})",
+            **self.basic_device_info,
+        )
+
+    @property
+    def basic_device_info(self) -> dict:
+        """Set basic device info."""
+
+        return {
+            "manufacturer": "Comelit",
+            "hw_version": "20003101",
```

Comment:
If the hardware version is fixed? Why is it here?

Reply:
As per my previous comment, Comelit refers to this device with a specific article number/hw version that won't change in future.